frontend/src/Movie/Index/Posters/MovieIndexPoster.js

```diff
@@ -253,9 +275,25 @@ class MovieIndexPoster extends Component {
             </div>
         }
 
+        {
+          showReleaseDate &&
+            <div className={styles.title}>
+              {getRelativeDate(
+                releaseDate,
+                shortDateFormat,
+                showRelativeDates,
+                {
+                  timeFormat,
+                  timeForToday: false
+                }
+              )}
```

Comment:
`{getRelativeDate(releaseDate, shortDateFormat, showRelativeDates, { timeFormat, timeForToday: false })}` for consistency with other usages

Reply:
I got this format from MovieIndexPosterInfo.js (used for sorting) - I am happy to change it but there seems to be a mixed consistency on this one already